conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
I see, so deleting run_docker_build.sh needs to stay.

I'll write a test and go from there.